internal/post/message.go

```diff
@@ -93,18 +105,26 @@ func (msg *Message) Bytes() []byte {
 			fmt.Fprint(sub, part.Content)
 		}
 
-		_ = wr.Close()
+		err := wr.Close()
+		if err != nil {
+			return nil, Error.Wrap(err)
+		}
 	// fallback if there are no parts, write PlainText with appropriate Content-Type
 	default:
 		fmt.Fprintf(&body, "Content-Type: text/plain; charset=UTF-8; format=flowed\r\n")
 		fmt.Fprintf(&body, "Content-Transfer-Encoding: quoted-printable\r\n\r\n")
 
 		enc := quotedprintable.NewWriter(&body)
-		_, _ = enc.Write([]byte(msg.PlainText))
-		_ = enc.Close()
+		if _, err := enc.Write([]byte(msg.PlainText)); err != nil {
+			return nil, Error.Wrap(err)
+		}
+
+		if err := enc.Close(); err != nil {
```

Comment:
Same reason than before, defer the close after the `enc` initialization.

Reply:
@ifraixedes Can I get a final review please? :) Thanks!

Reply:
Only this one is missing to be deferred after `enc := ...` in the line 102 using the `errs.Combine` as you used above.

Reply:
Oops missed that one. Good catch. Done!